### DISCUSSION TITLE
OSPO MindMap Introduction by Ana

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#94] [TODO & OSPO Alliance chapter]
Ana over at TODO Group is building an OSPO MindMap and is actively seeking collaborators.

https://www.openchainproject.org/featured/2022/05/15/webinar-41

I think this is a good project we can join, anyone interested?

**Comment by wangleo61** [comment]
I am very interesting this topic.

Leo

发件人: Will ***@***.***>
发送时间: Wednesday, May 18, 2022 6:28 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] OSPO MindMap Introduction by Ana (Discussion #94)


Ana over at TODO Group is building an OSPO MindMap and is actively seeking collaborators.

https://www.openchainproject.org/featured/2022/05/15/webinar-41

I think this is a good project we can join, anyone interested?

―

**Reply by will-ww** [reply]
可以来这个讨论，我们一起做起来：https://github.com/orgs/X-lab2017/discussions/35

**Comment by wangleo61** [comment]
好呀，谢谢王老师


发件人: Will ***@***.***>
发送时间: Wednesday, May 18, 2022 1:51 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] OSPO MindMap Introduction by Ana (Discussion #94)


可以来这个讨论，我们一起做起来：https://github.com/orgs/X-lab2017/discussions/35

―
